tests/handlers/test_oidc.py

```diff
@@ -599,18 +577,22 @@ def test_callback_session(self) -> None:
     )
     def test_exchange_code(self) -> None:
         """Code exchange behaves correctly and handles various error scenarios."""
-        token = {"type": "bearer"}
-        token_json = json.dumps(token).encode("utf-8")
-        self.http_client.request = simple_async_mock(
-            return_value=FakeResponse(code=200, phrase=b"OK", body=token_json)
+        token = {
+            "type": "Bearer",
+            "access_token": "aabbcc",
```

Comment:
Is this access token just garbage or does it match with something?

Reply:
It's garbage. We just check that `_exchange_code` returns the same dict later